src/test/kotlin/com/github/imflog/schema/registry/security/BasicAuthenticationIT.kt

```diff
@@ -391,6 +391,18 @@ class BasicAuthenticationIT {
                             .withBody(mapper.writeValueAsString(Schema(subject, 1, 1, "AVRO", emptyList(), schema)))
                     )
             )
+            wireMockAuthServerItem.stubFor(
+                WireMock.get(
+                    WireMock.urlMatching("/subjects")
+                )
+                    .withBasicAuth(USERNAME, PASSWORD)
+                    .willReturn(
+                        WireMock.aResponse()
+                            .withStatus(200)
+                            .withHeader("Accept", "application/json")
+                            .withBody(mapper.writeValueAsString(listOf("test-subject")))
+                    )
+            )
```

Comment:
With the changes mentioned above, you should be able to remove this part.

Reply:
Removed.